### ISSUE TITLE
Feature to add widgets to the container `FrostedGlass`

### ISSUE THREAD

**Issue by HeaTTheatR** [kivy-garden/frostedglass#1]
I've been testing the `FrostedGlass` widget for a while... Sure, this is a great job, but the widget itself has no practical use whatsoever. Because there is no way to place other widgets in it. Also, you cannot place other widgets OVER the `FrostedGlass` widget. Without this feature, I don't know how a user can use `FrostedGlass` in their application.

**Comment by DexerBR** [comment]
First of all, thanks for trying `FrostedGlass`.

You can certainly place other widgets OVER the `FrostedGlass` widget. Take a look at this simple example: https://github.com/kivy-garden/frostedglass#example
At the moment this is the correct way to use the widget (may change in the future).

Do:
```yaml
FrostedGlass:
    ...
FloatLayout:
    ...
    Buton:
        ...
```

Don't:
```yaml
FrostedGlass:
    ...
    FloatLayout:
        ...
        Buton:
            ...
```

However, the only thing that is not supported is to add child widgets, having `FrostedGlass` as a parent. Although it's not supported for now, I believe this is not a limiting factor for using it, as FrostedGlass is intended to be just a background (at least for now).
Maybe the README is missing some guidance, in relation to what I mentioned above, I will add more information as soon as I can.

**Comment by DexerBR** [comment]
Implemented in #2
